pkg/nsx/services/common/types.go

```diff
@@ -26,15 +26,22 @@ const (
 	TagScopeStaticRouteCRName       string = "nsx-op/static_route_cr_name"
 	TagScopeStaticRouteCRUID        string = "nsx-op/static_route_cr_uid"
 	TagScopeRuleID                  string = "nsx-op/rule_id"
+	TagScopeGoupID                  string = "nsx-op/group_id"
 	TagScopeGroupType               string = "nsx-op/group_type"
 	TagScopeSelectorHash            string = "nsx-op/selector_hash"
 	TagScopeNSXServiceAccountCRName string = "nsx-op/nsx_service_account_name"
 	TagScopeNSXServiceAccountCRUID  string = "nsx-op/nsx_service_account_uid"
+	TagScopeProjectID               string = "nsx-op/project_id"
+	TagScopeProjectGroupShare       string = "nsx-op/is_project_shared"
 	TagScopeNCPCluster              string = "ncp/cluster"
-	TagScopeNCPProject              string = "ncp/project"
-	TagScopeNCPVIFProject           string = "ncp/vif_project"
+	TagScopeNCPProjectUID           string = "ncp/project_uid"
+	TagScopeNCPVIFProjectUID        string = "ncp/vif_project_uid"
 	TagScopeNCPPod                  string = "ncp/pod"
 	TagScopeNCPVNETInterface        string = "ncp/vnet_interface"
+	TagScopeVPCProjectUID           string = "nsx-op/project_uid"     // Todo: pod subnet, need to change according to subnet controller
```

Comment:
I think we should use namespace instead of project
1. it is confusing with NSX Project
2. I think we will only support k8s

Reply:
Sure, Thanks for advice, we just keep project naming for NCP tags, for VPC, we change it in more appropriate naming in VPC context.